plugins/modules/ndo_vlan_pool.py

```diff
@@ -212,14 +204,13 @@ def main():
                 ops.append(dict(op="replace", path="{0}/{1}/description".format(path, match.index), value=description))
                 match.details["description"] = description
 
-            if module.params.get("vlan_ranges") is not None and match.details.get("vlan_ranges") != vlan_ranges:
+            if module.params.get("vlan_ranges") is not None and match.details.get("encapBlocks") != vlan_ranges:
                 ops.append(dict(op="replace", path="{0}/{1}/encapBlocks".format(path, match.index), value=vlan_ranges))
-                match.details["vlan_ranges"] = vlan_ranges
+                match.details["encapBlocks"] = vlan_ranges
```

Comment:
This is a breaking change. Why are we making this change? I thing we can keep vlan_ranges and add the encapBlocks one to the output so we don't break it.

Reply:
Done